Docs: update index.md based on UX feedback (#45)
intel-retail/automated-self-checkout: docs_src/index.md

# Automated Self Checkout

## Overview

> The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy Automated self-checkout using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured Automated self checkout pipeline that is optimized for Intel® hardware. Also included in is a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per vision stream.

Reviewer: Some re-wording suggestions

```suggestion
The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy self-checkout reference solution using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self checkout pipelines that are optimized for Intel® hardware.  Moreover, there also includes a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per video stream.
```

> The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy a self-checkout use case using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self checkout pipeline that is optimized for Intel® hardware.  The reference solution also includes a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per workload.

Reviewer 2: are you currently providing components to build/deploy for the self-checkout use case? or is it mostly benchmarking?

Author: It is a mix of both but most of it is tied to the benchmarking scripts

> The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy a self-checkout use case using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self checkout pipeline that is optimized for Intel® hardware.  The reference solution also includes a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per workload.

Reviewer 2: ```suggestion
The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy a self-checkout use case using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self-checkout pipeline that is optimized for Intel® hardware.  The reference solution also includes a set of benchmarking tools to evaluate the workload on different hardware platforms in order to minimize the cost per workload.
```

[![Automated Self Checkout Diagram](./images/vision-checkout-1.0.png)](./images/vision-checkout-1.0.png)

## Prerequisites

The following items are required to build the Automated Self Checkout solution. You will need to follow the guide that matches your specific hardware setup.

- [Ubuntu LTS Boot Device](https://ubuntu.com/tutorials/install-ubuntu-desktop#3-create-a-bootable-usb-stick)
- [Docker](https://docs.docker.com/engine/install/ubuntu/)
- [Git](https://git-scm.com/download/linux)

> ## Platform Installation and Pipeline Setup

Reviewer 2: This section is missing the pipeline setup link

Author: Updated to only point to the platform information which is the logical next step

> first verify that your platform is included on the [supported platform list](./platforms.md)

Reviewer: ```suggestion
First verify that your platform is included on the [supported platform list](./platforms.md)
```

> First verify that your platform is included on the [supported platform list](./platforms.md)

Reviewer 2: ```suggestion
First verify that your platform is included on the [supported platform list](./platforms.md).
```

> Once verified the setup steps for each supported hardware platform can be found [HERE](./hardwaresetup.md).

Reviewer 3: Replace "Once verified the setup steps..." with "Verify that the setup steps..."
